scripts/mirroring/.gitlab-ci.mirroring_example.yml

```diff
@@ -0,0 +1,42 @@
+# Example of .gitlab-ci.yml jobs
+# Use with scripts/mirroring/mirror_current_branch.sh
+
+stages:
+  - mirror
+
+.mirroring_template: &mirroring_template
+  stage: mirror
+  dependencies:
```

Comment:
Please describe this part in Read.me

Reply:
Added https://github.com/skilld-labs/skilld-docker-container/pull/239/files#diff-b21723e86f3dfcd0b209fd27e65791e5R23